patches/tModLoader/ReLogic/Content/Sources/IContentSource.cs.patch

```diff
@@ -54,6 +56,15 @@
 +		!Rejections.IsRejected(assetName) && GetExtension(assetName) != null;
 +
 +	// Added by TML.
-+	IEnumerable<string> GetAllAssetsStartingWith(string assetNameStart) =>
-+		EnumerateAssets().Where(s => s.StartsWith(assetNameStart));
++	[Obsolete, SuppressMessage("CodeQuality", "IDE0051:Remove unused private members")]
++	private IEnumerable<string> GetAllAssetsStartingWith(string assetNameStart)
++		=> GetAllAssetsStartingWith(assetNameStart, ignoreCase: false);
```

Comment:
Why make this private member? Are mods reflecting it?

Reply:
To avoid runtime breakage from introducing a new parameter. I don't recall the protection modifier mattering in that case.

Reply:
Without IgnoresAccessChecksTo, references to private members of another assembly will be thrown; one would need to annotate their mod assembly with `[assembly: IgnoresAccessChecksTo("tModLoader")]` (or module) to avoid this. Late, but relevant.